src/main/java/Aly/Aly.java

```diff
@@ -0,0 +1,451 @@
+package Aly;
+import Task.Task;
+import Task.Todo;
+import Task.Deadline;
+import Task.Event;
+
+import java.util.ArrayList;
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.Files;
+
+public class Aly {
+
+    //Constants
+    private static final Path SAVE_FILE = Paths.get("data","taskdata.txt");
+    private static final String RETURNING_TO_MAIN_MENU_MESSAGE = "Returning to main menu!";
+    private static final String RETURN_TO_MAIN_MENU_MESSAGE = "Enter 'exit' to return to main menu anytime!";
+    private static final String LOGO = "    _      _     _   _\n"
+            + "   / \\    | |   \\ \\ / /\n"
+            + "  / _ \\   | |    \\ V / \n"
+            + " / ___ \\  | |__   | |  \n"
+            + "/_/   \\_\\ |____|  |_|  \n";
+
+    //Prints line for clarity
+    private static final String LINE_SEPARATOR = "=".repeat(112);
+    private static void printLine() {
+        System.out.println(LINE_SEPARATOR);
+    }
+
+    //Initialising global variables/arrays
+    private static Scanner in = new Scanner(System.in);
+    private static ArrayList<Task> taskList = new ArrayList<>();
```

Comment:
Consider adding a new class for functions involving the list of tasks. Adding, Deleting and editing tasks can be done at a single place. This reduces the need to check for if list exists and if the index is within acceptable range at each add function.